crates/turbo-tasks-build/src/lib.rs

```diff
@@ -221,7 +236,9 @@ impl<'a> RegisterContext<'a> {
     }
 
     fn process_fn(&mut self, fn_item: ItemFn) -> Result<()> {
-        if has_attribute(&fn_item.attrs, "function") {
+        if has_attribute(&fn_item.attrs, "function")
+            && !ignored::task_ignored(&fn_item.sig.ident.to_string())
```

Comment:
A problem is probably that just ignoring the attribute is not enough. The attribute is also changing the signature, e. g. `async fn X() -> Result<Vc<Y>>` becomes `fn X() -> Vc<Y>`.

Reply:
Yeah I am going to see if it is possible to rewrite the signatures as well, have been messing around with it today to see if we can do that without requiring upstream tasks to change